### PR TITLE
chore(script): add bash script for macos

### PR DESCRIPTION
**Pull request by stavares843**
### What this PR does 📖

Adds a script with all the stuff that we need on macOS

Captura de ecrã 2024-01-18, às 22 15 06

Also updated the read me with the above script.

### CODE REVIEW

**Comment by WanderingHogan** [comment]
chances are you will probably have to chmod +x the script to run it. While this is a nice thought to include this, i think it is fine as-is.

File: macos-install_dependencies.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+
+# Install CMake
+brew install cmake
+
+# Install FFmpeg
+brew install ffmpeg
+
+# Install Opus
+brew install opus
```
Comment (dariusc93) [inline macos-install_dependencies.sh]: I havent checked but dont brew allow multiple dependencies to be installed at once or one at a time?

Comment (stavares843) [reply macos-install_dependencies.sh]: it does, i can update for that :joy:

Comment (stavares843) [reply macos-install_dependencies.sh]: done 🎉

**Comment by stavares843** [comment]
> chances are you will probably have to chmod +x the script to run it. While this is a nice thought to include this, i think it is fine as-is.



gonna check

**Comment by stavares843** [comment]
> chances are you will probably have to chmod +x the script to run it. While this is a nice thought to include this, i think it is fine as-is.

so we tested i asked phil to run the command and worked fine he didnt had to make the chmod + x

**Review by dariusc93 — APPROVED** [review]

**Pull request merged**